http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/KubernetesAppConfig.scala

```diff
@@ -139,8 +138,8 @@ final case class WorkflowsAppConfig(chartName: ChartName,
   def workflowsAppServices: Set[WorkflowsAppService] = services
```

Comment:
Consider if the `WorkflowsAppService` enum is needed anymore? It was added early on when we wanted to pick and choose which services were deployed via the `CROMWELL` app type. Nowadays, a given `WORKFLOWS_APP` will also include both `cbas` and `cromwell-reader` services, right? So maybe we don't need this level of granularity anymore.

Reply:
What are you suggesting to replace the usages of `workflowsAppServices` with?

Reply:
I didn't find all references.. but if it's just being used for `config.workflowsAppConfig.workflowsAppServices.contains(Cbas)`, etc… and it always contains the same list, we could just remove the method.

(Not a big deal, it was just added as a temporary thing which kept getting carried forward :))

Reply:
Made some changes in my latest commit, let me know if this looks good now?